samples/create_schedules.py

```diff
@@ -0,0 +1,57 @@
+####
+# This script demonstrates how to create schedules using the Tableau
+# Server Client.
+#
+# To run the script, you must have installed Python 2.7.9 or later.
+####
+
+import tableauserverclient as TSC
+import argparse
+import getpass
+import logging
+from datetime import time
+
+parser = argparse.ArgumentParser(description='Creates sample schedules for each type of frequency.')
+parser.add_argument('--server', '-s', required=True, help='server address')
+parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
+                    help='desired logging level (set to error by default)')
+args = parser.parse_args()
```

Comment:
The other samples probably need this as well -- but this should be in an `if __name__ == '__main__'` block so it only runs as a script, and so that we can check for no arguments and print the help.